build.gradle

```diff
@@ -15,12 +15,13 @@ import org.opensearch.gradle.testclusters.StandaloneRestIntegTestTask
 
 buildscript {
     ext {
+        job_scheduler_version = "1.3.1"
```

Comment:
Can we generate this value from the `opensearch_version` on line 20? The fewer places where we have to manually change during version bump, the better.